applications/produce_array_config.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application is a simple example of how to produce sim_telarray config \
```

Comment:
Remove 'simple' - what is 'simple' for you might be complex for others.

Reply:
Done!